public/components/trace_analytics/components/common/helper_functions.tsx

```diff
@@ -582,3 +589,56 @@ export const generateServiceUrl = (service: string, dataSourceId: string) => {
 
   return `${url}&datasourceId=`;
 };
+
+interface FullScreenWrapperProps {
+  children: React.ReactNode;
+  onClose: () => void;
+  isFullScreen: boolean;
+}
+
+export const FullScreenWrapper: React.FC<FullScreenWrapperProps> = ({
```

Comment:
could you add some comments above to explain what it does or why it's needed

Reply:
Added above
//EUI Data grid full screen button is currently broken, this is a workaround using overlay mask